sdk/communication/communication-phone-numbers/review/communication-phone-numbers.api.md

```diff
@@ -133,6 +142,38 @@ export interface SearchAvailablePhoneNumbersRequest extends PhoneNumberSearchReq
     countryCode: string;
 }
 
+// @public
+export class SipRoutingClient {
+    constructor(connectionString: string, options?: SipRoutingClientOptions);
+    constructor(endpoint: string, credential: KeyCredential, options?: SipRoutingClientOptions);
+    constructor(endpoint: string, credential: TokenCredential, options?: SipRoutingClientOptions);
+    deleteTrunk(fqdn: string, options?: OperationOptions): Promise<void>;
+    getRoutes(options?: OperationOptions): Promise<SipTrunkRoute[]>;
+    getTrunk(fqdn: string, options?: OperationOptions): Promise<SipTrunk | null>;
+    getTrunks(options?: OperationOptions): Promise<SipTrunk[]>;
```

Comment:
What is the max number of Trunks this operation can return? If it is unbounded, we may want to make this a pageable operation and name it `listTrunks` even if pagination is not implemented yet in the service, we should model this as pageable in the client to avoid breaking changes once the service supports paging

Reply:
Same for`getRoutes`

Reply:
There can be at most 250 trunks. 
When the API was designed, the board decided that pagination is not needed here. 
Same goes for routes.

Reply:
getTrunks was renamed to listTrunks
getRoutes was renamed to listRoutes